app/controllers/friend_requests_controller.rb

```diff
@@ -18,6 +19,7 @@ def create
   end
 
   def destroy
+    @user = @friend_request.receiver
```

Comment:
What did `destroy` need the `@user` for?

Reply:
Same as above

Reply:
Ok thanks for clarifying